config/projects/datadog-agent.rb

```diff
@@ -83,8 +83,10 @@
   # SysVInit service file
   if redhat?
     extra_package_file '/etc/rc.d/init.d/datadog-agent'
+    runtime_dependency 'libcap2'
   else
     extra_package_file '/etc/init.d/datadog-agent'
+    runtime_dependency 'libcap2-bin'
```

Comment:
Hehe... yeah, it's not `libpcap` but `libcap` - this is the `capabilities` library that would be needed to set `CAP_NET_RAW` and allow traffic sniffing without being root. So this isn't at all PCAP related.

That said, we could make it a **soft** dependency and just instruct the customer to install it if they want to enable `go-metro`. Might be the safer course of action.